registry/donor/views.py

```diff
@@ -237,3 +249,46 @@ def unignore_donor():
     else:
         flash("Při odebírání ze seznamu ignorovaných dárců došlo k chybě", "danger")
     return redirect(url_for("donor.show_ignored"))
+
+
+@blueprint.post("/override/")
+@login_required
+def save_override():
+    form = DonorsOverrideForm()
+    delete = "delete_btn" in request.form
+
+    if form.validate_on_submit():
+        if not delete:
+            # Save the override
+            override = DonorsOverride(**form.field_data)
+            db.session.merge(override)
+            db.session.commit()
+
+            DonorsOverview.refresh_overview()
+            flash("Výjimka uložena", "success")
+        else:
+            # Delete the override
+            override = DonorsOverride.query.get(form.rodne_cislo.data)
+            if override is not None:
+                db.session.delete(override)
+                db.session.commit()
+
+                DonorsOverview.refresh_overview()
+                flash("Výjimka smazána", "success")
+            else:
+                flash("Není co mazat", "warning")
+    else:
+        flash_errors(form)
+
+    return redirect(url_for("donor.detail", rc=form.rodne_cislo.data))
+
+
+@blueprint.get("/override/all")
+@login_required
+def get_overrides():
+    overrides_dict = {}
+
+    for override in DonorsOverride.query.all():
+        overrides_dict[str(override.rodne_cislo)] = override.to_dict()
```

Comment:
Není náhodou rodné číslo vždycky string?

Reply:
Je (předtím jsem myslel, že je to db.Column).